LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -71,6 +71,18 @@ public void ActivateAccounts()
                 Account account = pendingAccountStack.Pop();
                 account.Status = Status.Active;
 
+                if (account.Role == Role.Student)
+                {
+                    Student student;
+                    studentDictionary.TryGetValue(account.Id, out student);
```

Comment:
Do not assume that the dictionary contains the value, check for nulls. Only add to the active student list if student != null. Do the same null checking for active instructors

Reply:
Done